apps/website/next.config.ts

```diff
@@ -260,11 +261,18 @@ const config: NextConfig = {
     {
       source: "/wolf",
       destination: "/institute",
-      permanent: true,
+      permanent: false,
```

Comment:
Flipped this to temporary as /wolf feels generic enough we might want it for something else down the road (e.g. a wolf-specific page within ARRI)